source/install-guide/logging.rst

```diff
@@ -0,0 +1,38 @@
+.. _installing-index:
+
+Logging
+#######################
+
+This section describes how to use the geonetwork log files to find more details on incidents.
+
+Customising the log file location
+---------------------------------
+
+Log files are written to the generic log location of the container. It is possible to change the 
+location of the logfiles with an environment parameter `-Dlog_dir=/var/tomcat/`. 
+The path should end with `/` and the log files will be created in /var/tomcat/logs/geonetwork.log.
```

Comment:
Use `file` directive:
```
The path should end with :file:`/` and the log files will be created in :file:`/var/tomcat/logs/geonetwork.log`.
```

Does the logic really insert `logs` as in your example? I would assume if `-Dlog_dir=/var/something` is used the contents would go into `/var/something` (and not `/var/something/logs`)...